https/formData.go

```diff
@@ -119,49 +128,64 @@ func formParamWriter(
 	return nil
 }
 
-func toMap(keyPrefix string, param any, option ArraySerializationOption) (map[string][]string, error) {
-	if param == nil {
+func (fp *formParam) IsMultipart() bool {
+	contentType := fp.headers.Get(CONTENT_TYPE_HEADER)
+	if contentType != "" {
+		return contentType != FORM_URLENCODED_CONTENT_TYPE
```

Comment:
Why is the check `contentType != FORM_URLENCODED_CONTENT_TYPE`? Shouldn't it be `contentType == FORM_URLENCODED_CONTENT_TYPE`?

Reply:
This check is taken from Csharps Core library. i dont have more context.